lib/new_relic/agent/instrumentation/rails3/action_controller.rb

```diff
@@ -88,7 +88,7 @@ class ActionController::Base
   @name = :rails30_view
 
   depends_on do
-    defined?(::Rails) && ::Rails::VERSION::MAJOR.to_i == 3 && ::Rails::VERSION::MINOR.to_i == 0
+    defined?(::Rails::VERSION) && ::Rails::VERSION::MAJOR.to_i == 3 && ::Rails::VERSION::MINOR.to_i == 0
```

Comment:
Should it be `defined?(::Rails::VERSION::MAJOR) && defined?(::Rails::VERSION::MINOR) && ...` ?